CHANGELOG.md

```diff
@@ -1,5 +1,26 @@
 # Changelog
 
+## 0.17.0 (unreleased)
+
+### Breaking
+- `get_file_hash` is removed, use `get_hash` instead. Arguments do not change
+
+### Other
+
+- Add `get_taxonomy_term` function
+- Add slugify.paths_keep_dates option
+- Add command to generate shell completions
+- Fix link generation to colocated assets other than images
+- Add `get_hash` Tera function
+- Minify CSS and JS embedded in HTML
+- Fix slow image processing
+- Fix current_url in taxonomy term
+- Add new flag 'port_append' to give the ability to remove port from base url
+- `config.markdown` is now available in templates
+- Add `preserve_dotfiles_in_output` option in the config
+- Replace libsass by a Rust implementation: [grass](https://github.com/connorskees/grass)
```

Comment:
This is a breaking change:

> grass is not a drop-in replacement for libsass and does not intend to be. If you are upgrading to grass from libsass, you may have to make modifications to your stylesheets, though these changes should not differ from those you would have to make if upgrading to dart-sass.

Reply:
I'll move it, i forgot some stuff like division changed.